SpringSecurityOauth2ProviderGrailsPlugin.groovy

```diff
@@ -470,6 +475,26 @@ class SpringSecurityOauth2ProviderGrailsPlugin {
         }
     }
 
+    private configureBasicAuthenticationFilter = { conf ->
+        // Should the basic authentication filter be registered in the filter chain
+        boolean registerBasicAuthFilter = conf.oauthProvider.registerBasicAuthFilter as boolean
+
+        if(registerBasicAuthFilter) {
+            // This defines the bean using the alternative constructor that sets ignoreFailure to true,
+            // as an alternative to the filter offered by the Spring Security Core plugin:
+            basicAuthenticationFilter(BasicAuthenticationFilter, ref('authenticationManager', ref('oauth2AuthenticationEntryPoint'))) {

```

Comment:
Should we instead reference the `oauth2AuthenticationManager` bean directly to avoid any conflicts if a user (or another plguin) installs a different `AuthenticationManager` implementation under the `authenticationManager` bean?